fastlane/lib/fastlane/actions/cocoapods.rb

```diff
@@ -68,6 +69,11 @@ def self.available_options
                                        description: "Use bundle exec when there is a Gemfile presented",
                                        is_string: false,
                                        default_value: true),
+          FastlaneCore::ConfigItem.new(key: :use_pod_update,
+                                       env_name: "FL_COCOAPODS_USE_POD_UPDATE",
+                                       description: "Use 'pod update' before 'pod install'",
```

Comment:
I'm not sure if this option is such a good idea in general, as just always running `pod update` could cause your build to break (e.g. because of API changes in your dependencies). 

I'm not opposed to adding this for power users who know what they are doing, but at the very least the description here needs to somehow reflect that this could be dangerous and is not recommended for Testflight/AppStore or enterprise builds.